app/models/case_tracker.py

```diff
@@ -1,17 +1,26 @@
 from uuid import UUID
-from sqlmodel import SQLModel, Field, JSON, Relationship
-from app.models.base import TableModelMixin
+from sqlmodel import Field, JSON, Relationship
+from app.models.base import TableModelMixin, BaseRequest, BaseResponse
 
 
-class CaseTrackerBase(SQLModel):
-    case_id: UUID = Field(foreign_key="cases.id", index=True)
+class CaseTrackerBase:
```

Comment:
Should we have Base classes at all? Or should we standardise on:
Model,
ModelRequest,
ModelResponse?

Reply:
The base class here is simply a wrapper for the common fields between the request and model. If we didn't have it then we would have to duplicate it's fields into the request and model.